app/common.php

```diff
@@ -14,12 +14,12 @@
     define('DB_NAME', ltrim($dbopts["path"], '/'));
 }
 
-if (file_exists('config.php')) {
-    require 'config.php';
+if (file_exists(__DIR__.DIRECTORY_SEPARATOR.'config.php')) {
+    require __DIR__.DIRECTORY_SEPARATOR.'config.php';
```

Comment:
To avoid a regression, the config file have to be loaded from the root directory of Kanboard, not from `app/config.php` (`..` is missing).

(fixed in commits after yours)